changelog.md

```diff
@@ -8,6 +8,17 @@ class: changelog
 
 {% capture changelog %}
 
+## Version 5.0.16 September 24, 2019
+* Added new `referrer_policy` setting to add the `referrerpolicy` attribute when loading scripts or stylesheets.
+* Added a slight background color to dialog tab links when focused to aid keyboard navigation.
+* Fixed media poster value not updating on change.
+* Fixed openlink was not registered as a toolbar button.
```

Comment:
This is kinda awkward wording but Idk what it should be...

Reply:
openlink not being registered as a toolbar button?